index.html

```diff
@@ -9,8 +9,8 @@
 </div>
 <div class="row">
 	<div class="col-lg-12 text-center">
-		<h2 class="section-heading">The only Node.js first Desktop Automation Library</h2>
```

Comment:
It's not the only Node.js Desktop automation library, but it's the only Node.js first Desktop automation library.